normandy/recipes/tests/test_models.py

```diff
@@ -819,8 +819,15 @@ def test_enable_rollback_enable_rollout_invariance(self):
 
     @pytest.mark.django_db
     class TestCapabilities:
-        def test_v1_marker_included_if_non_baseline_capabilities_are_present(self, settings):
-            recipe = RecipeFactory(extra_capabilities=["non-baseline"])
+        def test_v1_marker_included_only_if_non_baseline_capabilities_are_present(self, settings):
+            action = ActionFactory()
+            settings.BASELINE_CAPABILITIES |= action.capabilities
+
+            recipe = RecipeFactory(extra_capabilities=[], action=action)
+            assert recipe.capabilities <= settings.BASELINE_CAPABILITIES
+            assert "capabilities-v1" not in recipe.capabilities
```

Comment:
In general I would consider this two separate tests (if they have different factory setups), but that's just a preferential thing, this looks good 👍

Reply:
Yeah, I was on the fence about it. On the one hand they share some things (like the action set up) and both test independently can't fully verify the feature. On the other hand, there are two set ups, two assertions, so it should be two tests. If they were bigger I'd split them, but as is I think I prefer them as one.